apiary.apib

```diff
@@ -0,0 +1,397 @@
+                                            "name": "Severus Snape",
+                                            "email": "halfbloodprince@hogwarts.uk",
+                                            "phone": "020666666"
+                                        }
+                                    ]
+                                },
+                                "date": "23-01-2017"
+                            }
+                        ]
+                    }
+                }
+            ]
+        }
+
+## Get data about a stack [GET /project/$id]
+
+Get data about a specific stack
+
++ Response 200 (application/json)
+
+        {
+            "data": {
+                "type": "stacks",
+                "id": "CWP001",
+                "name": "CWP Stack 001",
+                "primary_domain": "mom.govt.nz",
+                "agency": "Minstry of Magic",
+                "size": "large",
+                "disaster_recovery": "active",
+                "created": "02-07-2014",
+                "environments": {
+                    "count": 2,
+                    "data": [
+                        {
+                            "type": "environments",
+                            "id": "1",
+                            "name": "prod",
+                            "build_sha": "d0bcf80381579a529957d77a6b0a69303afebe7e",
+                            "deployer": {
+                                "data": [
+                                    {
+                                        "type": "users",
+                                        "id": "2379",
+                                        "name": "Albus Dumbledore",
+                                        "email": "sherbert.lemon@hogwarts.uk",
+                                        "phone": "0203040506"
+                                    }
+                                ]
+                            },
+                            "date": "24-01-2017"
+                        },
+                        {
+                            "type": "environments",
+                            "id": "2",
+                            "name": "uat",
+                            "build_sha": "d0bcf80381579a529957d77a6b0a69303afebe7e",
+                            "deployer": {
+                                "data": [
+                                    {
+                                        "type": "users",
+                                        "id": "21279",
+                                        "name": "Severus Snape",
+                                        "email": "halfbloodprince@hogwarts.uk",
+                                        "phone": "020666666"
+                                    }
+                                ]
+                            },
+                            "date": "23-01-2017"
+                        }
+                    ]
+                }
+            }
+        }
+
+## List users [GET /project/$id/users]
```

Comment:
To be honest, for MVP, I'd *not* create `/naut/users/$id` endpoint, because you can get this information from `/naut/project/$name/team` already. I'd remove drop "deployers" key in both project and environment API calls, because how do you even know that people will be after deployers, and not release managers?